vignettes/intro.Rmd

```diff
@@ -27,6 +28,238 @@ uirender(uiinput(icon("dog"),
 ), width = "200px", height = "100px")
 ```
 
+## Create a Shiny App
+Let's create a Shiny app with `semanticPage()` and just a `h1` title.
 
-To define a new page use:
+```{r echo=TRUE, include=TRUE}
+ui <- semanticPage(
+  title = "My first page",
+  h1("My page")
+)
+
+server <- function(input, output, session) {}
+
+shinyApp(ui, server)
+```
+
+Now let's add sidebar layout `sidebar_layout()`.
+
+```{r echo=TRUE, include=TRUE}
+ui <- semanticPage(
+  title = "My first page",
+  h1("My page"),
+  sidebar_layout(
+    sidebar_panel(),
+    main_panel()
+    )
+)
+
+server <- function(input, output, session) {}
+
+shinyApp(ui, server)
+```
+
+Next let's fill in the main panel `main_panel()` with a Semantic UI segment containing 2 cards.
```

Comment:
I think we should keep calling it `Fomantic UI` at this point

Reply:
done